tests/test_scripts/test_solph/test_nonconvex_investment/test_nonconvex_investment.py

```diff
@@ -0,0 +1,328 @@
+# -*- coding: utf-8 -*-
+
+"""
+This test contains a NonConvexInvestFlow class.
+
+
+This file is part of project oemof (github.com/oemof/oemof). It's copyrighted
+by the contributors recorded in the version control history of the file,
+available from its original location
+oemof/tests/test_scripts/test_solph/test_nonconvex_investment/test_nonconvex_investment.py
```

Comment:
We moved away from this to explicitly naming authors in the "SPDX-FileCopyrightText".

Reply:
Do you have an example of the actual practice ?

Reply:
E.g. in https://github.com/oemof/oemof-solph/blob/daebfa4e2a75e80fb244f7b58a07c95bfb18d626/src/oemof/solph/_groupings.py#L15-L22

Reply:
Thanks, we will implement it then

Reply:
Done in [927eb02](https://github.com/oemof/oemof-solph/pull/826/commits/927eb022490b37f6e757fb75b5ac1e37a4b73fcf)